doc/developers/maintainer.rst

```diff
@@ -17,25 +27,54 @@ Before a release
    change log is reasonably well curated. Some tools for these tasks include:
 
    - ``maint_tools/sort_whats_new.py`` can put what's new entries into
-     sections.
+     sections. It's not perfect, and requires manual checking of the changes.
+     If the whats new list if well curated, it may not be necessary.
 
    - The ``maint_tools/whats_missing.sh`` script may be used to identify pull
      requests that were merged but likely missing from What's New.
 
-Preparing a bug-fix-release
-...........................
+Permissions
+...........
+
+The release manager requires a set of permissions on top of the usual
+permissions given to maintainers, which includes:
+
+- *maintainer* role on ``scikit-learn`` projects on ``pypi.org`` and
+  ``test.pypi.org``, separately.
+- become a member of the *scikit-learn* team on conda-forge.
+- *maintainer* on ``https://github.com/MacPython/scikit-learn-wheels``
 
-Since any commits to a released branch (e.g. 0.999.X) will automatically update
-the web site documentation, it is best to develop a bug-fix release with a pull
+Preparing a release PR
+......................
+
+Since any commits to an existing branch (e.g. 0.999.X) will automatically
+update the web site documentation, it is best to develop a release with a pull
```

Comment:
Is it true that any branch to the repo has an online documentation? I thought we'd have a way to curate which branches we want to have the docs for, like with ReadTheDocs

Reply:
AFAIK any `0.XX.X` branch has a documentation online. Any change to the `0.21.X` branch for example, will be there almost immediately. And to add a new release, you just create the branch, and it'll automatically be there.